projects/lib-project/src/lib/dock/components/certificates/certificates.component.html

```diff
@@ -223,7 +223,7 @@
                             <div class="flex">
                                 <mat-label class="mr-6"> {{ item.label}}</mat-label>
                                 <b class="range-value">{{range.value}}</b>
-                                <mat-icon class="cursor-pointer text-xl text-iconGray" 
+                                <mat-icon class="cursor-pointer pl-1 -mt-[0.3rem] text-xl text-iconGray enable-with-pointer-events" 
```

Comment:
why is this different in every places "-mt-[0.3rem]" ? keep it in one place and "-mt-" why like this ?